### PR TITLE
added pie animation

### PR DESCRIPTION
**Pull request by melkayam92**
Thank you for contributing a pull request. 

Please ensure that you have signed the [CLA](https://docs.google.com/forms/d/19LpBBjykHPox18vrZvBbZUcK6gQTj7qv1O5hCduAZFU/viewform).

- [x] I have signed the CLA

### CODE REVIEW

File: src/Pie.js [hunk 1]
```diff
@@ -45,6 +49,19 @@ export default class PieChart extends Component {
...
+
+    this._animationRefArray = {};
+    this._startAnimation = this._startAnimation.bind(this)
+    this._createAnimation = this._createAnimation.bind(this)
+    this._onAnimationFinished = this._onAnimationFinished.bind(this)
+    if (this._shouldAnim()) this._animationArray = new Array(props.data.length)
+    
+  }
+
+  _shouldAnim = () => this.props.animate.enabled && this.props.data.length > 1
```
Comment (marzolfb) [inline src/Pie.js]: `this.props.options.animate` instead of `this.props.animate`

File: src/Pie.js [hunk 2]
```diff
@@ -64,6 +81,46 @@ export default class PieChart extends Component {
...
+        if (props.fillOpacity === 1) sliceRef._finished = true
+      });
+  }
+
+  _startAnimation(){
+    Animated.sequence(
+      this._animationArray.map(a =>
+        Animated.timing(a, {
+          toValue: 1,
+          duration: this.props.animate.duration
```
Comment (marzolfb) [inline src/Pie.js]: `this.props.options.animate` instead of `this.props.animate`

**Review by marzolfb — COMMENTED** [review]
My apologies for the delay in reviewing. 

Thanks for wanting to contribute. If you would like to continue, could you please sign the CLA as instructed above? Thanks.

I have reviewed it and I was unable to make it work without modifications changing references of  `this.props.animate` to `this.props.options.animate` (as noted in review comments).

Also, it would probably be helpful to add an animated version of the pie chart in the example app (as a way of showing the option exists). Are you willing to add that?

**Comment by marzolfb** [comment]
Also of important note, I've added a notice to the project readme that this project is going to be archived on February 28th. If you still want to get this change in before it gets archived we can do that if you can make the requested changes happen fairly soon.

**Comment by melkayam92** [comment]
Hey,

Thanks for the response.
I will make those changes soon and will sign the CLA.

**Comment by marzolfb** [comment]
Thanks for signing the CLA and providing the example but I have one last minor request - could you add a link to the new example in the main menu of the example app? Thanks!

**Comment by melkayam92** [comment]
Done.

**Review by marzolfb — APPROVED** [review]

**Pull request merged**